client/my-sites/upgrades/checkout-thank-you/footer/index.jsx

```diff
@@ -0,0 +1,70 @@
+/**
```

Comment:
Gotcha. We'll address that in a different pull request though so it doesn't block the rest of our work on this page.